modules/ROOT/pages/6.4-release-notes.adoc

```diff
@@ -0,0 +1,1154 @@
+// TINY-9556
+
+In earlier versions of {productname}, the properties set for the `renderUI`, caused conflicting issues with the theme rendering load times. As a consequence, the `renderUI` would finish rendering before the {productname} theme `silver` would finish loading.
+
+To fix this bug, {productname}, made changes to the `renderUI` properties that allowed the theme type the to return a `Promise<RenderResult>` instead of `RenderResult`, which corrects the issues found previously. 
+
+In {productname} 6.4, all `renderUI` property loading issues have been resolved.
+
+=== The searchreplace modal would close incorrectly when clicking outside of the alert that pops up when no match is found
+// TINY-9443
+
+In previous versions of {productname}, the search replace modal would close incorrectly when clicking outside of the alert dialog. The expexted behaviour in previous versions was that an error dialog would open when **no <string> match** was found within the {productname} editor viewport. The identifying issue found was when the user would `click` outside of the error dialog, resulting in the `Find and Replace` modal to closes incorrectly.
+
+In {productname} 6.4, when a **no <string> match** occurs within the {productname} editor viewport, an error banner will render in the `Find and Replace` modal beneith the (Replace with) field.
+
+As a result the when the instance occurs where **no <string> match** is triggered, the user will be prompted with the following warning: *`Could not find the specified string`*.
+
+=== Color picker on toolbar would not update when changing forecolor or backcolor from menu
+// TINY-9439
+
+In {productname} 6.3, when the https://www.tiny.cloud/docs/tinymce/6/6.3-release-notes/#new-color_map_foreground-and-color_map_background-options-store-custom-colors-selected-using-the-text-color-and-background-color-menus[new color map] was introduced, a regression was found that affected the UI for *Color Swatch*.
+
+The regression meant the colors would not render inside the color palette to reflect the selected `color_map_foreground` and `color_map_background` stored. To correct this {productname}, added a register that would store the `last color` for each  specific `node | cell` that will hold this `value` for the color picker pallette.
+
+In {productname} 6.4, when the color picker in the `toolbar` is updated when changing either the `forecolor` or `backcolor` from the `menubar`.
+
+=== The onSetup api function would not run when defining custom group toolbar button
+// TINY-9496
+
+A bug was discovered that affected the custom group toolbar button `config` in {productname}, due to a missing `Event Behaviour`. As a consequence, the `onSetup` api function would not run as intended when defining a custom group toolbar `button`, resulting in the `console` returning `blank`.
+
+In {productname} 6.4, this bug was fixed by adding a `Event Behaviour` to the toolbar group button that now calls the `api` as intended.
```

Comment:
`Event Behaviour` is an Alloy concept, which is not a concern to the integrators

Reply:
```suggestion
A bug was discovered that affected the custom group toolbar button in {productname}, due to missing some event handlers. As a consequence, the `onSetup` api function would not run as intended when defining a custom group toolbar button.

In {productname} 6.4, this bug was fixed by adding a handler to the toolbar group button that now calls the `api` as intended.
```